test/data/testinit.js

```diff
@@ -19,7 +19,7 @@ var parentUrl = window.location.protocol + "//" + window.location.host,
 	];
 
 // see RFC 2606
-this.externalHost = "example.com";
+this.externalHost = "releases.jquery.com";
```

Comment:
Note that this will trigger a request to https://releases.jquery.com on every AJAX test run. But it's served by Fastly and the traffic will be waaay lower than the normal CDN one, so I hope we're fine. Thoughts, @Krinkle?

Reply:
That's fine, yeah.